src/register/Register.jsx

```diff
@@ -2,126 +2,161 @@ import React, { useState, useContext } from "react";
+        body: JSON.stringify(payload),
+      };
 
-            fetch(`${BASE_API}/register`, requestRegister)
-                .then(async response => {
-                    let data = await response.json();
-                    if (response.status_code === 201)
-                        return setResponseMessage(data.message);
-                    setResponseMessage(data.message);
-                })
-                .catch(() => {
-                    setResponseMessage(SERVICE_UNAVAILABLE_ERROR);
-                });
-        }
+      fetch(`${BASE_API}/register`, requestRegister)
+        .then(async (response) => {
+          let data = await response.json();
+          if (response.status_code === 201)
+            return setResponseMessage(data.message);
+          setResponseMessage(data.message);
+        })
+        .catch(() => {
+          setResponseMessage(SERVICE_UNAVAILABLE_ERROR);
+        });
     }
+  };
 
-    const handleTogglePasswordDisplay = e => {
-        setIsPasswordShown(e.target.checked)
+  const handleTogglePasswordDisplay = (e) => {
+    setIsPasswordShown(e.target.checked);
+  };
+
+  const validateName = (e) => {
+    if (e.target.value.trim() === "") {
+      setIsValidName(false);
+    } else {
+      setIsValidName(e.target.checkValidity());
+    }
+  };
+  const validateUsername = (e) => {
+    setIsValidUsername(e.target.checkValidity());
+  };
+  const validateEmail = (e) => {
+    setIsValidEmail(e.target.checkValidity());
+  };
+  const validatePassword = (e) => {
+    if (e.target.value.trim() === "") {
+      setIsValidPassword(false);
+    } else {
+      setPassword(e.target.value);
+      setIsValidPassword(e.target.checkValidity());
     }
+  };
+  const validateConfirmPassword = (e) => {
+    setConfirmPassword(e.target.value);
+    setIsValidConfirmPassword(
+      e.target.checkValidity() && e.target.value === password
+    );
+  };
 
-    const validateName = e => {
-        if(e.target.value.trim()===''){
-          setIsValidName(false);
-        }
-        else{
-          setIsValidName(e.target.checkValidity());
-        }
-    };
-    const validateUsername = e => {
-        setIsValidUsername(e.target.checkValidity());
-    };
-    const validateEmail = e => {
-        setIsValidEmail(e.target.checkValidity());
-    };
-    const validatePassword = e => {
-        if(e.target.value.trim()===''){
-          setIsValidPassword(false);
-        }
-        else{
-          setPassword(e.target.value)
-          setIsValidPassword(e.target.checkValidity());
-        }
-    };
-    const validateConfirmPassword = e => {
-        setConfirmPassword(e.target.value)
-        setIsValidConfirmPassword(e.target.checkValidity() && e.target.value === password);
-    };
+  return isAuth ? (
+    <Redirect to="/" />
+  ) : (
+    <div className="container">
+      <div class="row align-items-start">
+        <div class="col">
+          <img src="src/assets/images/side_image.jpg" alt="background" />
```

Comment:
instead of using image path and have the image saved inside images folder, you can use the githubusercontent link of an image as it was used on the [About.jsx line 12](https://github.com/anitab-org/bridge-in-tech-web/blob/develop/src/home/About.jsx).